src/legacy/core_plugins/console/public/src/sense_editor/editor.js

```diff
@@ -564,7 +563,8 @@ export default function SenseEditor($el) {
         const  esMethod = req.method;
         const  esData = req.data;
 
-        const elasticsearchBaseUrl = chrome.getInjected('elasticsearchUrl');
+        // TODO: set url from ui settings
+        const elasticsearchBaseUrl = 'http://localhost:9200';
```

Comment:
This is likely to _never_ work in the real world as most people won't be running their ES on the local machine,  so seems like a bad thing to return for that.  This would basically break the feature for most people.  Could we maybe scrub out basic auth credentials?

Reply:
👍 , reverted in 6c8f1f41c8b2a6cc01fbe6dc04fbc40bcd4d7a11